webrtc.html

```diff
@@ -524,26 +528,31 @@
 
           <li>
             <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            execute the function from the front of the queue.</p>
+          </li>
+
+          <li>
+            <p>When the promise returned by the function is settled,
+            remove the corresponding object from the
+            <code>operations</code> array. After removal, if the array
+            is non-empty, execute the first object queued.  After
+            execution, settle <code>p</code> with the result from the
+            promise returned by the function.</p>
```

Comment:
Yes, this is where I mentioned we can re-order the two steps based on what the consensus is on "deque first" vs. "settle first".

As for the promises text, how about this?

"fulfill or reject p with the value or reason from the promise returned by the function"